src/shogun2-web/src/main/java/de/terrestris/shogun2/rest/AbstractRestController.java

```diff
@@ -32,14 +34,21 @@
 	private AbstractCrudService<E> service;
 
 	/**
-	 * Find all Entities.
+	 * Find all entities.
 	 *
 	 * @return
 	 */
 	@RequestMapping(method = RequestMethod.GET)
 	public ResponseEntity<List<E>> findAll() {
-		return new ResponseEntity<List<E>>(this.service.findAll(),
-				HttpStatus.OK);
+		final List<E> resultList = this.service.findAll();
+
+		if (resultList != null && !resultList.isEmpty()) {
```

Comment:
Same is true below where you ar doing string magic to get the classname for logging. This can be evaluated later,I guess